src/js/utils/responsive.js

```diff
@@ -1,29 +1,24 @@
-export const getBreakpoint = (windowWidth, theme) => {
-  let result;
+export const getBreakpoint = (viewportWidth, theme) => {
   const sortedBreakpoints = Object.keys(theme.global.breakpoints).sort(
     (a, b) => {
       const first = theme.global.breakpoints[a];
       const second = theme.global.breakpoints[b];
-      if (!first) return 1;
-      if (!second) return -1;
-      if (!first.value) return 1;
-      if (!second.value) return -1;
+      if (!first || !first.value) return 1;
```

Comment:
I think this might sort differently. In the old code, `!second && !first.value` will return -1 but will return 1 here.

Reply:
gotcha